### PR TITLE
Lost Ethereal

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Lost Ethereal

## Game Information
- **Title**: Lost Ethereal
- **URL**: https://lucregames.itch.io/lost-ethereal

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [ ] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/lostethereal/Lost Ethereal.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+export PORT_32BIT="Y"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
```
Comment (Cebion) [inline ports/lostethereal/Lost Ethereal.sh:22]: this is done in pm_helper no?

File: ports/lostethereal/Lost Ethereal.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    rm -f gamedata/*.{dll,ini,exe}
+fi
+
+# Check for file existence before trying to manipulate them:
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+[ -f "./gamedata/game.win" ] && mv gamedata/game.win gamedata/game.droid
+[ -f "./gamedata/game.unx" ] && mv gamedata/game.unx gamedata/game.droid
+
+# Make sure uinput is accessible so we can make use of the gptokeyb controls
+$ESUDO chmod 666 /dev/uinput
```
Comment (Cebion) [inline ports/lostethereal/Lost Ethereal.sh:53]: done in pm_helper

File: ports/lostethereal/Lost Ethereal.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+$GPTOKEYB "gmloader" -c ./lostethereal.gptk &
+
```
Comment (Cebion) [inline ports/lostethereal/Lost Ethereal.sh:56]: missing pm_helper

File: ports/lostethereal/Lost Ethereal.sh [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+$ESUDO chmod +x "$GAMEDIR/gmloader"
+
+./gmloader game.apk
+
+$ESUDO kill -9 $(pidof gptokeyb)
+$ESUDO systemctl restart oga_events &
+printf "\033c" > /dev/tty0
```
Comment (Cebion) [inline ports/lostethereal/Lost Ethereal.sh:63]: Alle the lines above are not needed. Missing pm_finish

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/lostethereal/Lost Ethereal.sh (hunk 1 above)
Comment (tabreturn) [reply ports/lostethereal/Lost Ethereal.sh]: Doh! Had that port lying around a while (which was based on the old Frax template)

**Review by cdeletre — CHANGES_REQUESTED** [review]
lostethereal-fs8
A quick made cover if you want (no obligation ^^)

**Review by Cebion — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**